backend/src/main/kotlin/app/ehrenamtskarte/backend/regions/database/repos/RegionsRepository.kt

```diff
@@ -33,4 +37,28 @@ object RegionsRepository {
     fun updateDataPolicy(region: RegionEntity, dataPrivacyText: String) {
         region.dataPrivacyPolicy = dataPrivacyText
     }
+
+    fun findRegionByPostalCode(postalCode: String, projectId: EntityID<Int>): RegionEntity {
+        try {
+            val csvInput = FileReader("src/main/resources/import/plz_ort_bayern.csv")
```

Comment:
I think there's a better way to access files from resources. Maybe take a look at other places where we have done this (eg. for the backend configuration).

Reply:
used `Classloader.getSystemResources`